PolyEngine/Engine/Src/TransformComponent.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "ComponentBase.hpp"
+
+namespace Poly {
+	class TransformComponent : public ComponentBase
```

Comment:
add macro: class **ENGINE_DLLEXPORT** TransformComponent... This is required to export the component outside of the DLL.